app/src/pages/inside/projectSettingsPageContainer/content/elements/ruleList/ruleList.jsx

```diff
@@ -44,7 +45,9 @@ const PlainRuleList = ({
           actions={actions}
           onToggle={onToggle}
           disabled={disabled}
-          content={ruleItemContent && <Content item={item} />}
+          content={
+            ruleItemContent && <Content item={item} ruleItemContentProps={ruleItemContentProps} />
```

Comment:
I suggest to use shorter name within Content component for its props or even spread them as the verbose explanation for `ruleItemContentProps` is not required inside it.
```javascript
ruleItemContent && <Content item={item} contentProps={ruleItemContentProps} />
// or even better
ruleItemContent && <Content item={item} {...ruleItemContentProps} />
```